app/assets/stylesheets/main.scss

```diff
@@ -14,15 +14,15 @@ $color-cool-blue: #205493;
 $color-purple: #4c2c92;
 
 html {
-    font-size: 16px;
+    font-size: 62.5%;
 }
 
 body {
-    font-size: 1.0625rem;
```

Comment:
Would eliminating this `body` declaration be enough to solve the problem? Because for `p` it's doing 1.0625 * 1.0625 * 16px? But I guess that only pushes up the text to 18px